components/grid/selection/overview.md

```diff
@@ -86,17 +94,21 @@ See how the row selection modes work
 
 ## Notes
 
+## Selection in Template
+
+When using the Grid [Template](https://docs.telerik.com/blazor-ui/components/grid/templates) and you want to stop the Selection from being triggered you should add stopPropagation directive to the element
+
 ## Editing Modes
 
 In the [Incell EditMode]({%slug components/grid/editing/incell%}) selection can be applied only via a checkbox column. This is required due to the overlapping action that triggers selection and incell editing (clicking in the row).
 
 ## Observable Collections
```

Comment:
This section may now be unnecessary. It should be replaced (without keeping the title) with the information on data changes.

Reply:
Done